### ISSUE TITLE
Create ADO security report as an artifact

### ISSUE THREAD

**Issue by marian-craciunescu** [azsk/DevOpsKit#949]
## Title</br>
ADO security report should be downloadable and saved as an artifact.
## Steps to reproduce

https://azsk.azurewebsites.net/09-AzureDevOps%28VSTS%29-Security/Readme.html 
Install and ran  the pipeline
```PowerShell

```
## Expected behavior</br> 

I want to be able to publish as an artifact the logs and stuff exported in  
` C:\Users\ADmin\AppData\Local\Microsoft\AzSK.AzureDevOpsLogs\<Org>\<scan_date>`
## Actual behavior</br>
No artifact can be publish

**Comment by marian-craciunescu** [comment]
Or at least how can I programatically use the files tored in the extension storage
```` 
Sending scan report to extension storage
Completed sending scan report to extension storage
`````

**Comment by marian-craciunescu** [comment]
@ritika-msft you seem the person to contact

**Comment by abhaydaga** [comment]
Scan reports can now also be downloaded from the build pipeline logs. This feature is available in the latest version (v.0.9.11).

**Comment by marian-craciunescu** [comment]
can you detail a litle more on how you can download the scan report?
Just parse the whole output log ? @abhaydaga

**Comment by marian-craciunescu** [comment]
i'm using AzSK.AzureDevOps Version: 0.9.12  but I can't download those said logs

**Comment by abhaydaga** [comment]
Go to the pipeline instance. Click the the three vertical dots at the top right corner. Refer the image below for reference

![image](https://user-images.githubusercontent.com/14292983/83869147-c05d1b80-a749-11ea-9342-3d41c2ac096a.png)

**Comment by marian-craciunescu** [comment]
Let me rephrase.I want to be able to download the full report  of the scan.
It previous version it was a `.csv` file  saved somewhere here.

Well for the new version there is no such file being generated.
The build logs  are not enough.You can't see just from there if a particular SVT passed /failed was it a manual/automatic check and , what's the impact and so on
````
The CSV file and LOG file are generated under a org-specific sub-folder in the folder
%LOCALAPPDATA%\Microsoft\AzSK.AzureDevOpsLogs\Org_[yourOrganizationName]
E.g.
C:\Users\UserName\AppData\Local\Microsoft\Azure.DevOpsLogs\Org_[yourOrganizationName]\20181218_103136_GADS
````

**Comment by marian-craciunescu** [comment]
Downloading the raw log I only get this kind of information
````
2020-06-05T13:29:52.9885752Z Checking resource [28/49] 
2020-06-05T13:29:53.1403006Z ================================================================================
2020-06-05T13:29:53.1403882Z Starting analysis: [FeatureName: Project] [ParentGroupName: companyName] [ResourceName: ProjectName] 
2020-06-05T13:29:53.1404606Z --------------------------------------------------------------------------------
2020-06-05T13:29:53.1829848Z Checking: [Project]-[Ensure that project visibility is set to private]
2020-06-05T13:29:53.3424998Z Checking: [Project]-[Scope of access of all pipelines should be restricted to current project.]
2020-06-05T13:29:53.7337008Z --------------------------------------------------------------------------------
2020-06-05T13:29:53.7338649Z Completed analysis: [FeatureName: Project] [ParentGroupName: companyName] [ResourceName: ProjectName] 
2020-06-05T13:29:53.7339549Z ================================================================================
2020-06-05T13:29:53.7347080Z  
````
